draft-ietf-ccwg-ratelimited-increase.md

```diff
@@ -284,6 +284,75 @@ A DCCP Congestion Control ID (CCID) specifing TCP-like behaviour ought to follow
 The text in {{Section 5.1 of ?RFC4341}} is updated by this document to specify the management of the
 cwnd when the sender is rate-limited.
 
+# An Example using cwnd represented in bytes
+
+The following informative example is provided for a sender that maintains the cwnd in bytes. This example uses a simplified calculation for the cwnd increase in slow start that can inflate the cwnd by not more than the number of bytes acknowledged in a single ACK.
+
+30 packets are sent in this example in three bursts of packets.
+The initial sender state is:
+  MSS=1000 bytes
+  cwnd=10000 bytes
+  maxFS=10000 bytes
+  ssthresh is infinity, i.e. the congestion control algorithm is in slow start.
+  One ACK is generated for each 2*MSS received bytes.
+
+Send 4000B MSS=1000, cwnd =10000
+  Send 1000  FS=1000; maxFS=10000
+  Send 2000  FS=2000; maxFS=10000
+  Send 3000  FS=3000; maxFS=10000
+  Send 4000  FS=4000; maxFS=10000
+Received 2 ACKs; maxFS=10000, if (cwnd<2*maxFS) {cwnd +=ACK’ed}
+  ACK for 2000 ACK’ed=2000; maxFS=10000, cwnd+=2000; cwnd=12000
+  ACK for 4000 ACK’ed=2000; maxFS=10000, cwnd+=2000; cwnd=14000
+Note: cwnd was increased, because FS used more than one half of cwnd.
```

Comment:
I don't understand the Note. FS was only 4000 which is *less* than half of cwnd. And why does this "half of cwnd" matter?